src/components/ListInput.tsx

```diff
@@ -0,0 +1,44 @@
+import styled from 'styled-components'
```

Comment:
In `HeaderLink.tsx` you used `styled-components/macro`, I guess you should use it everywhere or nowhere? 
Also, did you have to get a plugin so that `/macro` works? I tried it once, without any configuration, and it said that typescript doesn't know/can't find `/macro`.

Reply:
Right! Will add `/macro`. I just installed `@types/styled-components` and it worked 🤷

Reply:
Weird 😄